### PR TITLE
Add missing dependencies for the gene ontology query notebooks

### PR DESCRIPTION
**Pull request by bramalingam**
Adding a couple of python libraries to the docker file for the Gene Ontology notebooks.
Not sure if the best way is to update them here or have a requirements file in idr-py repository to manage the dependencies. Can close this PR if that sounds like a stable option.

@joshmoore @manics

### CODE REVIEW

**Comment by joshmoore** [comment]
Don't see any particular issues here; waiting on travis.

File: Dockerfile [hunk 1]
```diff
@@ -20,13 +21,14 @@ RUN conda install --name python2 --quiet --yes \
...
-    python-igraph
+    python-igraph 
 
 # RISE: "Live" Reveal.js Jupyter/IPython Slideshow Extension
 # https://github.com/damianavila/RISE
 RUN conda install --name python2 --quiet --yes -c bioconda zeroc-ice && \
     conda install --name python2 --quiet --yes -c damianavila82 rise && \
-    pip2 install py2cytoscape
+    conda install --name python2 --quiet --yes -c pdrops pygraphviz -y && \
+    /opt/conda/envs/python2/bin/pip install py2cytoscape pydot graphviz tqdm gseapy
```
Comment (manics) [inline Dockerfile]: Minor point on code-style: when you have multiple packages can you put them on separate lines (as in the rest of this file) to make future diffs clearer?
Edit: also alphabetical order

**Review by manics — CHANGES_REQUESTED** [review]

**Comment by joshmoore** [comment]
:+1: for the update strategy. @manics: you'd target this for prod44 (as `0.4.0`)?

**Comment by manics** [comment]
yes.

**Comment by joshmoore** [comment]
0.4.0 tag of `origin/develop` pushed.

**Pull request merged**